android/build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         mavenLocal()
         google()
         jcenter()
-        reactNativeFromUnpkg(it, reactNativeVersion)
+        packageFromUnpkg(it, "react-native", reactNativeVersion)
```

Comment:
Why this change? I think react-native is the only package we get from Unpkg

Reply:
I tried to use Hermes from Unpkg but the artifact itself have non-Maven structure so it was not unfortunatelly possible. I guess in future there may be some other packages that could be downloaded from Unpkg thus the "generification" of this utility function.

Reply:
Ok, in the end it does not make a big difference to me, I will leave it as is 😊